app/torii-providers/stripe.js

```diff
@@ -0,0 +1,29 @@
+import Oauth2 from 'torii/providers/oauth2-code';
+import { configurable } from 'torii/configuration';
+import { inject as service } from '@ember/service';
+
+export default Oauth2.extend({
```

Comment:
Extend the stripe provider provided by torii. Just extend. Don't copy paste what is there. That will lead to code duplication. We only want to change one property there.